### ISSUE TITLE
Cherry Popper Ice-cream

### ISSUE THREAD

**Issue by ChanceTheDice** [InZidiuZ/legacy-assets-issue-tracker#655]
**Name:**
Who is suggesting this? Please include discord name.
In city: D1-c3
Discord: ChanceTheDice#9874

**Summary:**
Write a short summary of what your suggestion is about.

I want to suggest turning the burger restaurant in mirror park that's called Horny's into a Cherry Popper Icecream parlor. 

**Link:**
If there are any links to what you're suggesting, please add them here.

**Reason:**
Why would your suggestion be a good addition to Legacy?

It makes sense to add a variety of food to the city and this also offers more places for your citizens to choose to work and eat. On top of that, I've expressed this idea to multiple people in the city already and every single person has expressed excitement at the idea!

The location makes sense being in a middle class area where they would be able to afford actually buying icecream but also to attract customers who wouldn't think them self too fancy for something so basic. Even if the idea for icecream is rejected, which I hope it isn't, the existing floor plan could be used for a variety of other business

**Comment by ChanceTheDice** [comment]
I see someone has been assigned and labels were added but I just want to check if I need to do anything or if it's still being considered

**Comment by MetaHatter** [comment]
The assignment just adds it to the queue to be looked at.  Note that the queue is LONG and it will likely take quite a while to review it.

**Comment by antihero6713** [comment]
Yes I am only 1 person that is very backed up and no laxatives. It will be noted. We did have another dev do a icecream shop
 on vinewood but i need to see all the info here.

**Comment by ChanceTheDice** [comment]
> Yes I am only 1 person that is very backed up and no laxatives. It will be noted. We did have another dev do a icecream shop on vinewood but i need to see all the info here.

Totally fine if you have one already made then please use it. It would be a waste not to use something you worked hard on. I would really like to use the Cherry Popper logo though

**Comment by GodyFromDiscord** [comment]
Can you create a concept logo for what you would like for branding, 512 x 512

**Comment by ChanceTheDice** [comment]
Cherry Popper Icecream already has a logo in the game but the image is 439 x 515. Can you use this or should I draw something original? (I do dabble in digital art so it wouldn't be that hard for me)

**Comment by GodyFromDiscord** [comment]
> Cherry Popper Icecream already has a logo in the game but the image is 439 x 515. Can you use this or should I draw something original? (I do dabble in digital art so it wouldn't be that hard for me)

I can use the original logo if I can find it, your choice

**Comment by ChanceTheDice** [comment]
![TheCherryPopperIceCreamCompany-Logo](https://user-images.githubusercontent.com/110052454/192175068-b1038b39-457b-45ed-b742-8beeedcef41b.svg)

**Comment by ChanceTheDice** [comment]
That's all I could find. If this doesn't work I can easily redesign it, you'll just have to let me know

**Comment by antihero6713** [comment]
Gonna close this for now. No needs for another food place
